examples/templates/gcp-devcontainer/main.tf

```diff
@@ -6,16 +6,49 @@ terraform {
     google = {
       source = "hashicorp/google"
     }
+    envbuilder = {
```

Comment:
curious: will we support the original form without `envbuilder`? if so, should we add a new example like `gcp-devcontainer`?

Reply:
The `envbuilder` provider only gets used if `cache_repo` is set.